packages/expo-sqlite/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "expo-sqlite",
+  "version": "1.0.0",
+  "description": "SQLite module",
+  "main": "build/index.js",
+   "types": "build/index.d.ts",
```

Comment:
Prettify this file, `types` is indented one space more than `main` and `scripts` contents are dedented. 🤷‍♂️ 🙂